src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -92,9 +92,12 @@ mutable struct Env
     finalize_called::Bool
     attached_models::Int
 
+    #Inner constructor
```

Comment:
```suggestion
```

Reply:
Sorry about this one, it was for me only and got pushed....